### ISSUE TITLE
Demos

### ISSUE THREAD

**Issue by petty** [eqcss/eqcss#21]
http://codepen.io/tomhodgins/pen/oLbXxG

... that one should be:

@element p and ...

as is it just starts out limegreem for the button that I assume you wanted to change when lots of text was inserted

Overall, sounds cool, especially for truncation; but adding debounce or throttle mechanism to help performance might be a good idea

**Comment by tomhodgins** [comment]
Hi @petty! Thanks for your interest. If you look at all of the feature demos [on one page](http://elementqueries.com/demos/features) you can see how the `min-lines` and `max-lines` demos work in context with all the other demos. The main thing to illustrate is that the demo changes after a certain number of lines are contained within the element :)

As for a debounce/throttle mechanism - I'm happy to let you know that we have that in place already! `EQCSS.apply()` will recalculate the styles immediately when called, while `EQCSS.throttle()` can be used as well and is throttled. The events that `EQCSS.js` automatically watches use `EQCSS.throttle()` by default, so performance shouldn't be an issue using the default settings, though we allow you to manually trigger recalculations exactly when you need to as well.

**Comment by petty** [comment]
So check this code pen out:

http://codepen.io/tomhodgins/pen/oLbXxG

That CSS should be 

/\* Max-lines */
@element p and (max-lines: 5) {
  .maxlines {
    background: greenyellow;
    border-color: limegreen;
  }
}

... then editing the text or loading with text that exceeds 5 lines changes the color.

**Comment by petty** [comment]
What does this demo do:

http://elementqueries.com/demos/flyout.html

... in Chrome I don't see it do anything, from the link off of Github for your project.

Maybe it only works locally??

**Comment by petty** [comment]
Other demos (again from your project page on Github) only work after resizing after the window loads:

http://elementqueries.com/demos/blocker.html#

**Comment by tomhodgins** [comment]
Hi @petty! I've recorded a video of me using the `max-lines` demo so you can see how it works!  A couple things to note:
- the div id set to `contenteditable` so you can interact with it and add lines
- there is no explicit `p` tag inside that element.

So no changes are needed to this demo for it to work. Here's the video of me interacting with it:

![max-lines](https://cloud.githubusercontent.com/assets/955601/16840525/248bb692-49a3-11e6-9b43-1ebc261fcf9d.gif)

As for the scrolling demos where the scroll is set on `body` or `html` — those demos will be working again here shortly. Previously (when those demos were built) we were calling `EQCSS.apply()` on scroll events, but that's unnecessary for performance reasons if we don't need to be watching it - so we switched it around so that only elements with `m**-scroll-*` conditions were watched for scroll with `EQCSS.throttle()`. The issue you're noticing is that browsers think about `body` and `html` elements differently than other elements, so we're working on a fix for this which will watch `window` instead for scroll if the element is `body` or `html`. You can check out progress toward that fix in Issue #20 here, and a fix should be released for this soon :D

In the meantime I've added this to the flyout and scroll blocker demos so they'll work as originally intended:

```
  <script>window.addEventListener('scroll',EQCSS.throttle)</script>
```
